### DISCUSSION TITLE
Change “Time played” of game

### DISCUSSION THREAD

**Discussion by DaraMac** [lutris/lutris#5172] [Ideas]
Piggybacking off this forum post: https://forums.lutris.net/t/change-time-played-of-game/8192
It would be handy if Steam games (or others) automatically had the time already played under the Steam launcher set, but if not then a simple fix is just a quick way to adjust the time played.

So far I've been manually editing the SQLite database which does work but is a bit cumbersome.

**Comment by strycore** [comment]
There is a edit widget for the playtime but I've disabled it because it was simply too user hostile (making users do maths to enter their playtime). Once the widget is properly written, it can be re-enabled.

**Reply by DaraMac** [reply]
Hey thanks that sounds good, just what I'd be looking for really.

**Comment by strycore** [comment]
For Steam games, it's pretty easy to get Steam's playtime.

**Reply by DaraMac** [reply]
Do you mean Lutris automatically getting it on import/installation or just reading it yourself?

**Reply by strycore** [reply]
Lutris automatically reading it. The API request we're making to Steam to get the user's library contains the playtimes if I remember correctly.

**Reply by DaraMac** [reply]
I hadn't actually noted that so I'll take a look thanks.
